blockchain_publisher/src/main/java/org/cardanofoundation/lob/app/blockchain_publisher/service/event_publish/LedgerUpdatedEventPublisher.java

```diff
@@ -79,7 +78,7 @@ public void sendReportLedgerUpdatedEvents(String organisationId,
                         val cardanoFinalityScoreM = reportEntity.getL1SubmissionData().flatMap(L1SubmissionData::getFinalityScore);
                         val ledgerDispatchStatus = blockchainPublishStatusMapper.convert(publishStatusM, cardanoFinalityScoreM);
 
-                        return new ReportStatusUpdate(reportEntity.getReportId(), ledgerDispatchStatus);
+                        return new ReportStatusUpdate(reportEntity.getReportId(), ledgerDispatchStatus, reportEntity.getL1SubmissionData().flatMap(L1SubmissionData::getTransactionHash));
```

Comment:
Here you put list of blockchain receipts but it will always be list of one receipt for now ("CARDANO_L1") as type.